contracts/upgradeable_contracts/erc20_to_erc20/BasicForeignBridgeErcToErc.sol

```diff
@@ -39,7 +39,7 @@ contract BasicForeignBridgeErcToErc is BasicBridge, BasicForeignBridge {
         return bytes4(keccak256(abi.encodePacked("erc-to-erc-core")));
     }
 
-    function claimTokens(address _token, address _to) public onlyIfOwnerOfProxy {
+    function claimTokens(address _token, address _to) public onlyIfUpgradeabilityOwner {
```

Comment:
Do we still need this renaming even if we know that it will be removed under https://github.com/poanetwork/poa-bridge-contracts/pull/203?

Reply:
You are right, probably it just make sense to remove it for `BasicForeignBridgeErcToErc.sol` 
 and `ForeignBridgeErcToNative.sol`